mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -107,9 +109,20 @@ def SetDestroy(self):
             self._commandsocket.SetDestroy()
         if self._configurationsocket is not None:
             self._configurationsocket.SetDestroy()
-    
+
     def _ExecuteCommand(self, command, fireandforget=False, timeout=2.0, recvjson=True, checkpreempt=True, blockwait=True):
-        # type: (Dict, bool, float, bool, bool, bool) -> Optional[Dict]
+        # type: (Dict, bool, float, bool, bool, bool) -> Any
```

Comment:
Why is this Any? This will throw errors when the other functions that do `return self._ExecuteCommand` have a type hint returning Optional[Dict].

Reply:
It is Any, because depending on `recvjson` value it may return not dict. We could also be Union[Dict, str, None] . But its actually the opposite of what you say - with Any Pylance doesnt show any errors, while with Union[Dict, str, None] or even Optional[Dict] we would have to do bunch of type checks. E.g. for return value not being None before doing `SendAndReceive(...).get('parametervalue', None)` (example from another client).

Of course ideally we would like to not use Any unless we have to. But the way our code is written makes it quite hard for now. And _ExecuteCommand is private function - so users of the client wont interact with it anyway.